src/react-components/src/components/Link/Link.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from "react";
+import bem from "../../utils/bem";
+import { uid } from "react-uid";
+import { LinkTypes } from "./LinkTypes";
+import { element } from "prop-types";
+import Icon from "../Icons/Icon";
+import { iconRotationTypes } from "../Icons/IconTypes";
+
+export interface LinkProps {
+    /** Controls the link visuals—action, button, or default. */
+    linkType?: LinkTypes;
+
+    /** Href attribute */
+    href?: string;
+
+    /** BEM specifiers */
+    modifiers?: string[];
+
+    /** BEM specifiers */
+    blockName?: string;
+
+    /** Additional attributes, such as rel=nofollow, to pass to the <a> tag */
+    attributes?: {};
+}
+
+export default function Link(props: React.PropsWithChildren<LinkProps>) {
+    const {
+        href,
+        modifiers,
+        blockName,
+        attributes,
+        linkType = LinkTypes.Default,
+    } = props;
+
+    const passedProps = { href, attributes };
+    passedProps.href = props.href;
+    passedProps.attributes = props.attributes;
+
+    let link_base_class = "link";
+
+    let childProps = {};
+
+    let icon;
+
+    let elementChildren = React.Children.map(
+        props.children,
+        (child: React.ReactElement) => {
+            if (typeof child === "string" && !props.href) {
+                throw new Error("Link needs prop 'href'");
+            } else if (child.type === "a") {
+                childProps = child.props;
```

Comment:
This is also why that one link didn't have an underline... wasn't getting its `href`